applications/northstar/main.tf

```diff
@@ -35,6 +35,15 @@ variable "name" {
   description = "The application name."
 }
 
+variable "storage_name" {
+  description = "The storage bucket name."
+}
+
+variable "backup_storage_bucket" {
+  description = "Optionally, the bucket to replicate storage backups to."
+  default     = null
+}
```

Comment:
This is a variable we used to provide to Rogue to allow us to configure our cross-region backup bucket in production. Since this bucket is now living under Northstar, we provide this variable here instead!